### PR TITLE
Bugfix/tg 229 unpopulated sensors

### PR DESCRIPTION
**Pull request by velmyk**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-backend/pull/70" target="_blank">ReviewNinja

File: src/API/dashboard/dashboard.test.js [hunk 1]
```diff
@@ -19,11 +19,14 @@ describe('## Dashboard APIs', () => {
...
-                })
-                .catch(done);
+            DashboardModel.create({
+                devices: [
+                    { device: device._id, hidden: false },
+                    { device: '5825941220788d3c52e7766c', hidden: false}
+                ]
+            })
+            .then(() => done())
+            .catch(() => done());
```
Comment (ashilay) [inline src/API/dashboard/dashboard.test.js]: we have the same result for both cases, let use finally?

File: src/API/dashboard/dashboard.test.js [hunk 2]
```diff
@@ -44,11 +47,20 @@ describe('## Dashboard APIs', () => {
...
                     done();
                 });
         });
+
+        it('should filter unpopulated sensors', done => {
+            request(app)
+                .get('/api/dashboard')
+                .expect(httpStatus.OK)
+                .then(res => {
+                    expect(res.body.devices.length).equals(1);
```
Comment (ashilay) [inline src/API/dashboard/dashboard.test.js]: It is better to set result to const here and expect it below, out of "then" callback

File: src/config/env/test.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 export default {
     env: 'test',
     db: 'mongodb://localhost/db',
-    port: 3000,
+    port: 3001,
```
Comment (disist) [inline src/config/env/test.js]: Why you decided that it's need to be changed?

Comment (velmyk) [inline src/config/env/test.js]: Server works on same (3000). I changed it to watch test and run server simultaneously.

**Pull request merged**